packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -143,7 +145,7 @@ $filter-pill-ease: ease-in-out;
 }
 
 .c-filterPill--selected {
-    box-shadow: 0 0 0 2px $color-focus;
+    box-shadow: 0 0 0 1px $filter-pill-selected-color;
 }
```

Comment:
Does this want to be reduced to 1px as well as the colour changing?

Reply:
I'm unsure! This was a change @1itvinka asked me to add in this PR - I will try to find out

Reply:
@AlixRichards there will be another PR here shortly to deliver the styling - if we need to tweak to 2px I can add the change there if that's cool :)